packages/docs/command-line-interface/account.md

```diff
@@ -357,21 +357,24 @@ _See code: [packages/cli/src/commands/account/lock.ts](https://github.com/celo-o
 
 ### New
 
-Creates a new account locally using the Celo Derivation Path (m/44'/52752'/0/0/indexAddress) and print out the key information. Save this information for local transaction signing or import into a Celo node. Ledger: this command has been tested swapping mnemonics with the Ledger successfully (only supports english)
+Creates a new account locally using the Celo Derivation Path (m/44'/52752'/0/changeIndex/addressIndex) and print out the key information. Save this information for local transaction signing or import into a Celo node. Ledger: this command has been tested swapping mnemonics with the Ledger successfully (only supports english)
 
 ```
 USAGE
   $ celocli account:new
 
 OPTIONS
+  --addressIndex=addressIndex
```

Comment:
Flagging this to make sure we update the celo holders guide as well.

Reply:
There were a few refs in the CLI code/docs I missed so thanks for the callout but this doesn't appear in any docs. Are you confusing it with the `ledgerCustomAddresses` flag?